.github/workflows/mise.yml

```
@@ -0,0 +1,36 @@
+name: mise
+on: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)